server.py

```diff
@@ -19,14 +18,14 @@ def endpoint():
     print("Character endpoint reached.")
     print("---------------------------")
 
-    req_dict = request.get_json()
+    req = request.get_json()
 
     try:
-        database = CallDatabase(MONGO_DOCKER_IP, MONGO_DOCKER_PORT, "/app")
+        database = CallDatabase('localhost', MONGO_PORT, "/app")
```

Comment:
This makes it work wheen running locally at. the expence of docker changes needed in the future. it's a trade off, probably a good one for right now.